ui/src/overridden/frontsite/Document/DocumentDetails/DocumentPanel/StandardNumber.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Label } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+export const StandardNumber = ({ metadata, ...props }) => {
+  if (metadata.identifiers) {
+    const standardNumbers = metadata.identifiers.filter(
+      sn => sn.scheme === 'STANDARD_NUMBER'
+    );
+    if (standardNumbers[0]) {
```

Comment:
it might be an empty list:
```suggestion
    if (standardNumbers) {
```

You can also do:
```
return standardNumbers ? <> : null;
```